### PR TITLE
add aubo_robot to documentation index for distribution

### PR DESCRIPTION
**Pull request by liuxinwust**

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -466,6 +466,16 @@ repositories:
...
     status: maintained
+  aubo_robot:
+    doc:
+      type: git
+      url: https://github.com/auboliuxin/aubo_robot.git
+      version: master
+    source:
+      type: git
+      url: https://github.com/auboliuxin/aubo_robot.git
+      version: indigo
```
Comment (tfoote) [inline indigo/distribution.yaml]: There is no indigo branch on this repository. Looking at the repository there's only the `master` branch so this should be `master`

**Comment by tfoote** [comment]
This is redundant with #12747

**Comment by auboliuxin** [comment]
.ERROR: In file '''/home/travis/build/ros/rosdistro/indigo/distribution.yaml''': Could not validate source entry for repo aubo_robot with error [[[Could not validate repository with url https://github.com/auboliuxin/aubo_robot.git and version indigo from entry at line '''474''']]]

F

======================================================================

FAIL: test_function (test_url_validity.TestUrlValidity)

----------------------------------------------------------------------

Traceback (most recent call last):

File "/home/travis/build/ros/rosdistro/test/test_url_validity.py", line 258, in test_function

self.assertFalse(detected_errors)

AssertionError: [u"In file '''/home/travis/build/ros/rosdistro/indigo/distribution.yaml''': Could not validate source entry for repo aubo_robot with error [[[Could not validate repository with url https://github.com/auboliuxin/aubo_robot.git and version indigo from entry at line '''474''']]]"] is not false

-------------------- >> begin captured logging << --------------------

rosdistro: DEBUG: Load index from "file:///home/travis/build/ros/rosdistro/test/../index.yaml"

rosdistro: DEBUG: Load index from "file:///home/travis/build/ros/rosdistro/test/../index.yaml"

--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------

Ran 5 tests in 55.448s

FAILED (failures=1)

The command "nosetests -s" exited with 1.

Done. Your build exited with 1.

**Comment by tfoote** [comment]
There is no branch named indigo. In this context the "version" should be your source branch, which must be `master` on your repo since that's the only branch.

**Pull request closed without merging**